mk/generate.mk

```diff
@@ -49,7 +49,7 @@ clean/resources:
 POLICIES_DIR := pkg/plugins/policies
 COMMON_DIR := api/common
 
-policies = $(foreach dir,$(shell find pkg/plugins/policies -maxdepth 1 -mindepth 1 -type d | grep -v -e core | sort),$(notdir $(dir)))
+policies = $(foreach dir,$(shell find pkg/plugins/policies -maxdepth 1 -mindepth 1 -type d | grep -v -e core -e generator | sort),$(notdir $(dir)))
```

Comment:
Could you please place `generator` package inside the `pkg/plugins/policies/core`?